category-committed.php

```diff
@@ -0,0 +1,313 @@
+      </div>
+      <div class="flex-grid-item flex-item-xxl-12 text-align-center pt-10 pt-l-7 pt-s-12">
+        <h1 class="u-visuallyhidden">Committed</h1>
+        <h3 class="u-visuallyhidden">Would you go to prison for your politics?</h3>
+        <div class="committed-archive__logo"></div>
+        <div class="committed-archive__subtitle "></div>
+      </div>
+    </div>
+  </section>
+</div>
+  <section class="container">
+    <div class="flex-grid-row mt-4 mb-4">
+      <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-l-1 flex-item-l-10 flex-offset-xxl-2 flex-item-xxl-8">
+        <h3 class="mb-4 font-size-12 font-weight-bold">Listen to the trailer:</h3>
+        <iframe width="100%" height="166" scrolling="no" frameborder="no" allow="autoplay" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/1270698601&color=%23ffab70&auto_play=false&hide_related=true&show_comments=false&show_user=false&show_reposts=false&show_teaser=false"></iframe>
+      </div>
+    </div>
+    <div class="flex-grid-row mt-4 mb-5">
+      <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-l-1 flex-item-l-10 flex-offset-xxl-2 flex-item-xxl-8 font-serif committed__serif-large text-paragraph-breaks">
+        <p>In the 1970s, the Provisional IRA was in the early days of its armed campaign to end British rule on the island of Ireland. In the United States, a small group of activists began organising on their behalf. They called themselves the Irish Northern Aid Committee, or Noraid—and they were looking for a fight.</p>
+        <p>Hosted by documentary filmmaker Nate Lavey, Foreign Agent is a podcast series about the connection between ordinary Irish Americans and a revolutionary socialist guerrilla group. In six episodes, travelling back and forth across the Atlantic over three decades of conflict, Foreign Agent explores how regular Americans became militant advocates for the cause of Irish freedom.</p>
+        <p>It’s a story of guns smuggled in furniture and wild plots to build homemade missiles. But it’s also about the political and material power that the Irish American community wielded, and how the intoxicating spell of nationalism created a movement out of seemingly irreconcilable social and political positions. And at every step of the way, the US government tried to shut them down. This is the story of the Troubles—as seen through American eyes.</p>
+      </div>
+    </div>
+    <div class="flex-grid-row mb-6 mb-s-4">
+      <div class="flex-grid-item flex-item-s-12 flex-item-xxl-12 font-color-white committed-archive-listen__box">
+        <div class="committed-archive__box">
+          <div class="committed-listen-now-title mb-1"></div>
+          <div class="font-weight-bold committed-listen-now__links font-size-13 font-size-s-12">
+            <a href="https://podcasts.apple.com/us/podcast/committed-the-iras-american-connection/id1624937065?uo=4">Apple Podcasts</a>,
```

Comment:
Obv these links will need to be updated before going live. I would think sensible to remove the def incorrect Foreign Agent ones just incase they accidentally get set live.

Reply:
I have a list of things to change, we don't have the links yet